web-services-app/src/main/webapp/WEB-INF/service-context.xml

```diff
@@ -498,6 +498,7 @@
         <property name="aclService" ref="aclService"/>
         <property name="repoObjLoader" ref="repositoryObjectLoader"/>
         <property name="basePath" value="${data.dir}/zip/"/>
+        <property name="fileLimit" value="500" />
```

Comment:
We still need to figure out what the limit should be. We can deploy this to test and try it out with works that have different numbers of files like listed in the files, and then adjust as needed